targets/kodi

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh -e
+# Copyright (c) 2016 The crouton Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+REQUIRES='x11'
+DESCRIPTION='Installs the KODI media player. (Approx. 140MB)'
+HOSTBIN='startkodi'
+CHROOTETC='kodi-keyboard.xml kodi-cycle.py'
+. "${TARGETSDIR:="$PWD"}/common"
+
+### Append to prepare.sh:
+if [ "$DISTRO" = 'debian' ]; then
+    echo "deb http://http.debian.net/debian jessie-backports main" | sudo tee /etc/apt/sources.list.d/kodi.list
+fi
+
+if [ "$DISTRO" = 'ubuntu' ]; then
+    sudo apt-get install -y software-properties-common
```

Comment:
Refer to this [old xfce target](https://github.com/dnschneid/crouton/blob/90ad47ff22e148db41ba627992ee5cc5d0c14175/targets/xfce#L13) for how to pull in PPAs.  We don't want to unnecessarily install things.